tox.ini

```diff
@@ -126,9 +126,10 @@ description =
 basepython = python3
 usedevelop = True
 changedir = doc/en
-deps = -r{toxinidir}/doc/en/requirements.txt
+deps =
+    -r{toxinidir}/doc/en/requirements.txt
```

Comment:
could you keep such formatting changes out?

Reply:
FIxed